src/nvim/eval.c

```diff
@@ -6733,6 +6733,39 @@ static void prepare_assert_error(garray_T *gap)
   }
 }
 
+/*
+ * Append "str" to "gap", escaping unprintable characters.
+ * Changes NL to \n, CR to \r, etc.
+ */
+static void ga_concat_esc(garray_T *gap, char_u *str)
+{
+    char_u *p;
+    char_u buf[NUMBUFLEN];
+
+    if (str == NULL) {
+	ga_concat(gap, (char_u *)"NULL");
+	return;
+    }
+
+    for (p = str; *p != NUL; ++p)
+	switch (*p) {
```

Comment:
need to fix the indent. Try `make lint` on your local machine to see any other lint issues.